### PR TITLE
Allow durations to be divided by other durations

### PR DESCRIPTION
**Pull request by eric-wieser**
For instance `Duration(secs=10) / Duration(secs=5) == 2`

### CODE REVIEW

**Comment by dirk-thomas** [comment]
Please update the docblocks to clarify that duration arguments are also supported.

File: src/genpy/rostime.py [hunk 1]
```diff
@@ -398,6 +398,8 @@ def __floordiv__(self, val):
             return Duration(self.secs // val, self.nsecs // val)
         elif t == float:
             return Duration.from_sec(self.to_sec() // val)
+        elif isinstance(val, Duration):
+            return self.to_sec() // val.to_sec()
```
Comment (eric-wieser) [inline src/genpy/rostime.py]: Note that unlike everything else mentioned in #49, `Duration // Duration` _is_ meaningful

**Comment by eric-wieser** [comment]
Docblocks are done - would you prefer I squash this?

**Comment by dirk-thomas** [comment]
Yes, please squash.

**Comment by eric-wieser** [comment]
(squashed)

**Comment by eric-wieser** [comment]
ICYMI, github now can squash PRs when merging

**Comment by dirk-thomas** [comment]
Thanks.

**Pull request merged**